packages/flutter/lib/src/physics/spring_simulation.dart

```diff
@@ -37,16 +44,26 @@ class SpringDescription {
     double ratio = 1.0,
   }) : damping = ratio * 2.0 * math.sqrt(mass * stiffness);
 
+  /// The greater the mass, the larger the amplitude of oscillation,
+  /// and the slower the speed of returning to the equilibrium position.
```

Comment:
Nit: I worry "speed" here is misleading, if I get myself back in the mindset of physics class. Maybe say "rate" instead? Or "the longer the time to return to the equilibrium position"

Reply:
Thanks for your reply, your concerns are very valid. I prefer the latter